model/actions/trigger_transaction.go

```diff
@@ -629,9 +629,27 @@ func (t *TransactionFilter) Validate(ctx ValidatorContext) (response ValidateRes
 		response.Merge(t.StateChanged.Validate(ctx.With("stateChanged")))
 	}
 
+	// Constraint check
+	response.Merge(t.ValidateConstraint(ctx))
+
 	return response
 }
 
+func (t *TransactionFilter) ValidateConstraint(ctx ValidatorContext) (response ValidateResponse) {
+	if t.From != nil || t.To != nil || t.Function != nil ||
```

Comment:
I think this validation should be done before trying to validate filters - to point in trying if none exists. Code readability would benefit from having `if` condition moved to a fn